test/controllers/experiments_controller_test.rb

```diff
@@ -1,10 +1,45 @@
 require 'test_helper'
 
-class ExperimentsControllerTest < ActionController::TestCase
+
+class ExperimeechntsControllerTest < ActionController::TestCase
+  tests ExperimentsController
```

Comment:
Dlaczego to tu musi być? W supervisorze bez czegoś takiego działają testy kontrolerów.